python/katana/local/entity_type.pyx

```diff
@@ -0,0 +1,31 @@
+from libc.stdint cimport uint8_t
+from libcpp.string cimport string
+
+from katana.cpp.libsupport.entity_type_manager cimport EntityTypeManager
+
+
+cdef class EntityType:
+    """
+    A class representing node/edge types.
+
+    Only works for atomic types. Intsersection type support TBD
+    """
+
+    @property
+    def type_id(self):
+        return self._type_id
+
+    cdef string type_name(self):
+        typename_option = self._type_manager.GetAtomicTypeName(self._type_id)
+        return typename_option.value()
```

Comment:
Is this needed to avoid issues with non-default constructable types? If so, document it in a comment. I know I haven't done that my self. Please call me out on that in the future so I fix it.

Reply:
I don't know to be honest. I fought with this forever. Eventually this configuration caused it to pass the tests, so I'm afraid to change it.

Reply:
Could you just add a comment like? "This function seems to be needed to make the terrible confusing Cython compiler happy."

Reply:
Okay, I think that change might not have been what mollified the cython compiler. I'm changing it to just one function in the next commit